bottlecap

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+
+# bottlecap, 'cause it's kinda like cork... get it?
+
+usage() {
+	echo "usage:"
+	echo "bottlecap [--dev] [--runtime $runtime] [--build-dir $build_dir]  [--container $container] assemblerargs..."
+	echo "  --dev: rebuild and reinstall mantle, the build scripts, and the ostree-releng-scipts before running coreos-assembler"
+	echo "  --runtime: (auto) specify whether to use podman or docker. Defaults to podman if it is in \$PATH"
```

Comment:
It might be better to say "Defaults to $runtime" and just let the default value from the script automatically be substituted in.

Reply:
Only concern there is someone looks at it on one system and thinks "Oh, the default is X", then writes some script for another system with that in mind and gets a different result.

Reply:
yeah, i could go either way. if we have to worry about people hacking on the script then all bets are off.